srcpkgs/vala-panel-appmenu/template

```diff
@@ -0,0 +1,79 @@
+# Template file for 'vala-panel-appmenu'
+pkgname=vala-panel-appmenu
+version=0.7.1
+revision=1
+build_style=cmake
+configure_args="-DENABLE_XFCE=ON -DENABLE_VALAPANEL=ON
+ -DENABLE_MATE=ON -DENABLE_JAYATANA=ON -DENABLE_APPMENU_GTK_MODULE=ON
+ -DJAVA_INCLUDE_PATH=/usr/lib/jvm/openjdk/include
+ -DJAVA_JVM_LIBRARY=/usr/lib/jvm/openjdk/lib
+ -DJAVA_AWT_LIBRARY=/usr/lib/jvm/openjdk/lib/amd64"
+hostmakedepends="pkgconf"
+makedepends="vala-devel bamf-devel gtk+-devel gtk+3-devel libglib-devel
+ libxkbcommon-devel openjdk libpeas-devel libdbusmenu-glib-devel cmake-vala
+ xfce4-panel-devel xfconf-devel libmate-panel-devel libXt-devel"
+short_desc="Global Menu for Vala Panel"
+maintainer="John Eira <john.eira@web.de>"
+license="LGPL-3.0-or-later"
+homepage="https://gitlab.com/vala-panel-project/vala-panel-appmenu"
+distfiles="https://gitlab.com/vala-panel-project/vala-panel-appmenu/uploads/c0f85b42a2a228ad8c3413357e02fb28/vala-panel-appmenu-${version}.tar.xz"
+checksum=54a1b4f6ceca63eb3b79facc4327424bdd82f0fe3b7c6139326dbe0ec0824dcc
+
+appmenu-gtk-module-devel_package() {
```

Comment:
it should depend on all other packages and in the main package. use version restriction.

Reply:
fixed